src/helpers/Environment.ts

```diff
@@ -195,4 +195,44 @@ export class EnvHelper {
     }
     return zapPool;
   }
+
+  /**
+   * Indicates whether the give feature is enabled.
+   *
+   * The feature is enabled when:
+   * - REACT_APP_GIVE_ENABLED is true
+   * - give_enabled parameter is present
+   *
+   * @param url
+   * @returns
+   */
+  static isGiveEnabled(url: string): boolean {
+    const giveEnabled = EnvHelper.env.REACT_APP_GIVE_ENABLED;
+    const giveEnabledParameter = url && url.includes("give_enabled");
+
+    if (giveEnabled || giveEnabledParameter) return true;
+
+    return false;
+  }
```

Comment:
is this feature flag going to be kept alive or killed before launch?

Reply:
If you're good with it being live at the time we merge into develop/master, then we can kill it. Or else we can just modify the Fleek deployment to have a true value in the environment variables. Less code to edit.